progress-reports/icon-bridge/September-2022-BTP-Progress-Update.md

```diff
@@ -0,0 +1,37 @@
+## Intro
+
+This is Icon Bridge by Web3Lab. This is progress report from 1-September-2022 to 30-September-2022
+
+## Summary
+
+Have been working on the increasing test coverage level of the BMR module.
+Have been working on the SNOW integration.
+
+## Deliverables
+
+### Accomplished this cycle
+
+__Deliverables ready__
+
+| Name                                                                                | Development State | Notes | Source / location                                           |
+|-------------------------------------------------------------------------------------|-------------------|-------|-------------------------------------------------------------|
+| Improve BMR code unit tests coverage                                                | In Progress       |       | Epic https://github.com/icon-project/icon-bridge/issues/396 |
+| Create BMR unit test for receiver functionality                                     | Released          |       | 1) https://github.com/icon-project/icon-bridge/issues/399   |
+| Refactor the icon relay receiver to make it testable                                | Released          |       | 2) https://github.com/icon-project/icon-bridge/issues/614   |
+| Integrate ICE/SNOW to the BTP                                                       | In Progress       |       | Epic https://github.com/icon-project/icon-bridge/issues/500 |
+| Test start BTP with ICE chain                                                       | Completed         |       | 1) https://github.com/icon-project/icon-bridge/issues/501   |
```

Comment:
This is the ICON Bridge, not BTP. BTP is distinct in that verification happens on-chain instead of on the relay.